### ISSUE TITLE
Integration Experience (Platform Spike Team) access to DEV @ GovCloud-US

### ISSUE THREAD

**Issue by considerable** [department-of-veterans-affairs/va.gov-team#36033]
## Description
_What details are necessary for understanding the specific work or request tracked by this issue?_
Platform Spikes Integration Experience Team needs a "home" in VA development environment to deploy newly-made DEV-grage microservices for testing and POC. Those can be DEV instanced of old EC2 style like current VETS-API and of new K8s style like PgHero deployed in EKS. 

## Background/context 
_Any additional context for the reader to know, if applicable (optional, please delete if unused)_ 
It is in the nature of the Spike to make proof of concept, try, error, repeat so that we need a place in VA's DEV of GovCloud-US to deploy our work.
The big monolith vets-api that is powering va.gov is going to transition into a collection of miscroservices in EKS as mentioned in those epics linked to this ticket. Integration Experience Team will be piloting and making those microservices for EKS that are very hard to "try at home" w/o actual EKS sandbox. Furthermore, the new microservices are likely to be shipped for production not in a big bulk but one after another in a slow pace as they become ready.
One pilot microservice that came to EKS before us is PgHero. The first one in our current sprint is SideKiq UI, and we are blocked by having no place to complete development and testing.
Please help us figure what do we need to get the job well done so that our workflow can be reused later by other teams.

## Technical notes
_Notes around work that is happening, if applicable (optional, please delete if unused)_

- This is something we may try in the DEV environment as the EKS equivalent of "hello world"
https://aws.amazon.com/premiumsupport/knowledge-center/eks-cluster-kubernetes-dashboard/

- We need to be able to use CI/CD tools for the workflow alike deployment of PgHero too

- It would be nice to have EKS stats sent out by the Datadog Agent per https://docs.datadoghq.com/agent/kubernetes/?tab=daemonset

- VA repo from where we will be deploying our new microservices-in-the-making into our DEV @ VA environment is https://github.com/department-of-veterans-affairs/vets-api-monitoring-suite

- The first project in that repo will be SideKiq UI

---
## Tasks
- [ ] _What things need to happen?_

- The IET group policy created
- The IET engineers access granted
- The training/walkthrough session conducted

## Acceptance Criteria 
- [ ] _What are the specific things that must be true in order for this to be accepted?_

- (optionally) official K8s dashboard aka "Hello World" deployed
- SideKiq UI deployed from IET repo vets-api-monitoring-suite
- (optionally) newly deployed activity viewed from DataDog or other monitoring tool
 
---
### Reminders
- [ ] Please attach your team label and any other appropriate label(s) (operations, devops, and needs-grooming will automatically be applied as part of the template)
- [ ] Please connect to an epic (this will typically be done by the Platform Operations PM or TL)

**Comment by considerable** [comment]
@CosmicSyntax @ewilson-adhoc @rogersarkis @jjulian @mleclerc00 @ericboehs

**Comment by td-usds** [comment]
I am fine with access to DEV at this point, put me in the loop when other environments are needed as well...

**Comment by jhouse-solvd** [comment]
We are tracking work to onboard an application to platform infrastructure for the IET here: 

https://github.com/department-of-veterans-affairs/va.gov-team/issues/36811

It's possible that this issue can be closed in favor of the above.

**Comment by ewilson-adhoc** [comment]
This ticket can indeed be marked as a duplicate. Closing it out.
